frontend_lib/src/component/Timeline/Comment.styl

```diff
@@ -23,8 +23,6 @@
     &__text
       margin-left 20px
       margin-top 10px
-  p
-    margin 0
-    & img
+  p img
       max-width 100%
```

Comment:
the tab is right here? it looks a little big

Reply:
Indeed, I forgot to deindent this line

Reply:
```suggestion
  p img
    max-width 100%
```